bedrock/mozorg/urls.py

```diff
@@ -15,7 +15,10 @@
     page('about/manifesto', 'mozorg/about/manifesto.html'),
     page('about/manifesto/details', 'mozorg/about/manifesto-details.html'),
     page('about/leadership', 'mozorg/about/leadership.html'),
-    page('about/policy/lean-data', 'mozorg/about/policy/lean-data.html'),
+    page('about/policy/lean-data', 'mozorg/about/policy/lean-data/index.html'),
```

Comment:
I believe this breaks a few URL reverse lookups for `url('mozorg.about.policy.lean-data')`, so you may need to update on pages that link to the lean data landing page.

Reply:
Oh good call, knew I'd forget some details in the rush...